### ISSUE TITLE
Debugger not working on Windows 11

### ISSUE THREAD

**Issue by SebStreb** [Dman95/SASM#183]
The debugger doesn't seem to be working on Windows 11.

With this code:
```
%include "io.inc"

section .text
global CMAIN
CMAIN:

mov ebp, esp; for correct debugging
    
;write your code here
GET_UDEC 4, ebx
PRINT_UDEC 4, ebx
   
 xor eax, eax
 ret
```

I get the following error when launching the debugger (executing works fine): 
```
[07:24:02] Build started...
[07:24:02] Built successfully.
[07:24:02] Debugging started...
unknown register: 00x004013ed
```

Windows version: Windows 11 Éducation
SASM version: 3.12.2

**Comment by piper4hire** [comment]
It’s also not working on my Commodore 64.

**Comment by Dman95** [comment]
Fixed in SASM version 3.13.2. Thank you!
